src/docs/automation-and-resiliency/request-quota-adjustment-for-openshift-project-set.md

```diff
@@ -216,14 +216,14 @@ This will increase the chances of us approving your request quickly (or helping
 > 
 > **Outline the current and desired states of the relevant OpenShift objects.**
-> My frontend app pods use 250m CPU limit each. Currently, the HPA can spin up 6 of these before I hit the CPU limit quota for my namespace. I don't know how many pods the HPA would attempt to spin up if given more quota, so I'm not sure how much CPU limit I should expect to use. For now, I am requesting a single step increase in my CPU quota, and will revisit if I continue to encounter this problem.
+> My frontend app pods use 100m CPU request each. Currently, the HPA can spin up 6 of these before I hit the CPU request quota for my namespace. I don't know how many pods the HPA would attempt to spin up if given more quota, so I'm not sure how much CPU request I should expect to use. For now, I am requesting a single step increase in my CPU quota, and will revisit if I continue to encounter this problem.
 >  
 > **What steps have you taken to fit your application into your current quota?**
-> I tried running my frontend app pods with 150m CPU limit instead, but that caused problems with the application during high load, because the existing pods weren't able to handle load spikes temporarily while the HPA was spinning up new pods. I don't want to try to reclaim CPU limit from my database pods because they're only running at 100m CPU limit, and it's not best practice to run a database pod with very low limits.
+> I tried running my frontend app pods with 80m CPU request instead, but that caused problems with the application during high load, because the existing pods weren't able to handle load spikes temporarily while the HPA was spinning up new pods. I don't want to try to reclaim CPU request from my database pods because they're only running at 100m CPU request, and it's not best practice to run a database pod with very low requests.
 
 This request is likely to get approved with just a few follow-up questions from the Platform Services Team. Even though it doesn't provide a clear picture of how the namespace's resource usage should look in the future, the requester has explained why they lack that information. They've also made a reasonable request for a single-step increase, planning to request more only if there's evidence they need it.
 
-If the request includes statistics on the resource usage of the front-end pods, confirming the explanation about the load problems caused by running the pods with a lower CPU limit, this might be one of those rare CPU requests that could be approved without any further questions from the Platform Services Team!
+If the request includes statistics on the resource usage of the front-end pods, confirming the explanation about the load problems caused by running the pods with a lower CPU request, this might be one of those rare CPU requests that could be approved without any further questions from the Platform Services Team!
  
```

Comment:
I think we also should add a section to give some idea of how the auto approve will happen.